src/dioptra/restapi/v1/plugins/controller.py

```diff
@@ -112,7 +112,7 @@ class PluginIdFilesEndpoint(Resource):
     @accepts(PluginFileGetQueryParameters, api=api)
     @responds(schema=PluginFilePageSchema, api=api)
     def get(self, id: int):
-        """Gets the PluginFile resources for a Plugin resource."""
+        """Gets a list of al PluginFile resources for a Plugin resource."""
```

Comment:
Small typo, should read `"""Gets a list of all PluginFile resources for a Plugin resource."""`